instance/templates/instance/ansible/secret-keys.yml

```diff
@@ -14,9 +15,14 @@ NOTIFIER_LMS_SECRET_KEY: '{{ secret_generator.NOTIFIER_LMS_SECRET_KEY }}'
 PROGRAMS_SECRET_KEY: '{{ secret_generator.PROGRAMS_SECRET_KEY }}'
 ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_KEY : '{{ secret_generator.ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_KEY }}'
 ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_SECRET : '{{ secret_generator.ECOMMERCE_SOCIAL_AUTH_EDX_OIDC_SECRET }}'
-ECOMMERCE_EDX_API_KEY: '{{ secret_generator.EDXAPP_EDX_API_KEY }}' # Same as the edxapp API key
 EDXAPP_ANALYTICS_API_KEY: '{{ secret_generator.EDXAPP_ANALYTICS_API_KEY }}'
 FORUM_API_KEY: '{{ secret_generator.FORUM_API_KEY }}'
-EDXAPP_COMMENTS_SERVICE_KEY: '{{ secret_generator.FORUM_API_KEY }}'
-NOTIFIER_COMMENT_SERVICE_API_KEY: '{{ secret_generator.NOTIFIER_COMMENT_SERVICE_API_KEY }}'
-NOTIFIER_USER_SERVICE_API_KEY: '{{ secret_generator.NOTIFIER_USER_SERVICE_API_KEY }}'
+
+# Keys that need to match other keys.  The variables will be filled in by Jinja2 template
+# processing in Ansible rather than by the Django template processor, hence the verbatim block.
+{% verbatim %}
+ECOMMERCE_EDX_API_KEY: '{{ EDXAPP_EDX_API_KEY }}'
```

Comment:
@smarnach Alternatively, you could also move the reference to the `secret_generator` out of the template, and simply set the values in the context - that might actually be the most readable way.